datafusion/core/tests/parquet/mod.rs

```diff
@@ -403,9 +437,13 @@ fn make_timestamp_batch(offset: Duration) -> RecordBatch {
         schema,
         vec![
             Arc::new(arr_nanos),
-            Arc::new(arr_micros),
-            Arc::new(arr_millis),
-            Arc::new(arr_seconds),
+            Arc::new(arr_nanos_timezoned),
+            Arc::new(arr_micros.clone()),
```

Comment:
I wonder if these really need to be cloned 🤔 

```suggestion
            Arc::new(arr_micros),
```

Reply:
Done.